README.md

```diff
@@ -20,28 +20,28 @@ By default, firectl searches `PATH` for the firecracker binary. The location of
 
 ```
 Usage:
-  firectl
+  firectl [OPTIONS]
 
 Application Options:
       --firecracker-binary=     Path to firecracker binary
-      --kernel=                 Path to the kernel image (default: ./vmlinux)
-      --kernel-opts=            Kernel commandline (default: ro console=ttyS0 noapic reboot=k panic=1 pci=off nomodules)
+      --kernel=                 Path to the kernel image
+      --kernel-opts=            Kernel commandline
       --root-drive=             Path to root disk image
       --root-partition=         Root partition UUID
       --add-drive=              Path to additional drive, suffixed with :ro or :rw, can be specified multiple times
       --tap-device=             NIC info, specified as DEVICE/MAC
       --vsock-device=           Vsock interface, specified as PATH:CID. Multiple OK
       --vmm-log-fifo=           FIFO for firecracker logs
-      --log-level=              vmm log level (default: Debug)
+      --log-level=              vmm log level
       --metrics-fifo=           FIFO for firecracker metrics
   -t, --disable-hyperthreading  Disable CPU Hyperthreading
-  -c, --ncpus=                  Number of CPUs (default: 1)
+  -c, --ncpus=                  Number of CPUs
       --cpu-template=           Firecracker CPU Template (C3 or T2)
-  -m, --memory=                 VM memory, in MiB (default: 512)
+  -m, --memory=                 VM memory, in MiB
       --metadata=               Firecracker Metadata for MMDS (json)
   -l, --firecracker-log=        pipes the fifo contents to the specified file
+  -s, --socket-path=            path to use for fircracker socket, defaults to a temporary file
   -d, --debug                   Enable debug output
-  -h, --help                    Show usage
```

Comment:
What is the reasoning about removing `--help` in the `README.md`?

Reply:
The help option is added dynamically by the library that does the arg parsing. when no args are passed we call into the library's [WriteHelp()](https://godoc.org/github.com/jessevdk/go-flags#WriteHelp) to print usage text in this case. I didn't notice that the output for this help text was different than when you run with the help flag.

I'm not sure if we are using the library incorrectly or this is just a weird case.